src/Types.ts

```diff
@@ -2,48 +2,56 @@ import { CanvasType, QRErrorCorrectLevel } from './Enums';
 
 export interface QRCodeConfig {
     binarizeThreshold?: string;
-    binarize?: boolean,
-    size: number,
-    margin: number,
-    typeNumber: number,
-    colorDark: string,
-    colorLight: string,
-    correctLevel: QRErrorCorrectLevel,
-    backgroundImage?: string,
-    backgroundDimming: string,
-    logoImage?: string,
-    logoScale: number,
-    logoMargin: number,
-    logoCornerRadius: number,
-    whiteMargin?: boolean,
-    dotScale: number,
-    autoColor?: boolean,
-    text: string,
-    maskedDots: boolean,
-    canvasType?: CanvasType,
+    binarize?: boolean;
+    size: number;
+    margin: number;
+    typeNumber: number;
+    colorDark: string;
+    colorLight: string;
+    correctLevel: QRErrorCorrectLevel;
+    backgroundImage?: string;
+    backgroundDimming: string;
+    logoImage?: string;
+    logoScale: number;
+    logoMargin: number;
+    logoCornerRadius: number;
+    whiteMargin?: boolean;
+    dotScale: number;
+    autoColor?: boolean;
+    text: string;
+    maskedDots: boolean;
+    canvasType?: CanvasType;
+    eyeBallShape?: string;
```

Comment:
Enums should be used instead of string for eyeBallShape,  eyeFrameShape, eyeBallColor,  eyeFrameColor.

Reply:
Refactored using Enums